# Description of configuration files can be found at "https://gitlab.ethz.ch/ics-group/projects/andrea/crs-2.0/-/wikis/ros-configuration"
state_type: pacejka_car
input_type: pacejka_car

# process_noise:
#   type: multivariate_gaussian
#   seed: 42
#   mean:
#     value: [[0], [0], [0], [0], [0], [0]]

sensors:
  sensor_names: ["vicon", "imu"]
  vicon:
    R:
      value: [[0.0000001], [0.00000001], [0.00000001]]
      is_diag: true
    frequency: 50 # Hz
    key: vicon # optional, if not set us e name of sensor
    measurement_noise:
      type: multivariate_gaussian

  imu:
    R:
      value: [[0.00000001], [0.00000001], [0.0000001]]
      is_diag: true
    frequency: 50 #Hz
    key: imu # optional, if not set us e name of sensor
    measurement_noise:
      type: multivariate_gaussian
      seed: 1
      mean:
        value: [[0], [0], [0]]

initial_state: [0, -1, 0, 0.5, 0, 0]
initial_input: [0.0, 0.2]

frequency: 100 #Hz, speed of simulator

# Not yet implemented
input_delay: 0.01 # input delay in seconds
do_collision_checks: false
